api/app/settings/common.py

```diff
@@ -645,6 +646,26 @@
     },
 }
 
+REDIS_CACHE_NAME = "redis-cache"
```

Comment:
Should the redis cache be used generically like this? Or should the name of the cache represent what is stored in it? (so we might end up with multiple caches using the same redis cache URL).

Reply:
Yes, good shout, I have removed it to allow user to decide which backend they want to use (limited to rate limiting for now)